microsoft/testsuites/core/timesync.py

```diff
@@ -94,14 +94,28 @@ def verify_timesync_ptp(self, node: Node) -> None:
         assert_that(dmesg.get_output()).contains(self.ptp_registered_msg)
 
         # 2. PTP device name is hyperv.
+        # In some Linux VMs you may see multiple PTP devices listed.
+        # One example is for Accelerated Networking the Mellanox mlx5 driver also
+        # creates a /dev/ptp device. Because the initialization order can be different
+        # each time Linux boots,the PTP device corresponding to the Azure host might
+        # be /dev/ptp0or it might be /dev/ptp1.
+        # To determine which PTP device corresponds to the Azure host, you can check the
+        # clock_name file. Atleast one of the PTP devices should have
+        # a clock_name of 'hyperv'.
```

Comment:
```
        # In some Linux VMs you may see multiple PTP devices listed.
        # One example is for Accelerated Networking the Mellanox mlx5 driver also
        # creates a /dev/ptp device. Because the initialization order can be different
        # each time Linux boots, the PTP device corresponding to the Azure host might
        # be /dev/ptp0 or it might be /dev/ptp1.
        # To determine which PTP device corresponds to the Azure host, you can check the
        # clock_name file. At least one of the PTP devices should have
        # a clock_name of 'hyperv'.
```

Reply:
Fixed typo